src/swagger-specs/streaming-ingestion.yml

```diff
@@ -0,0 +1,405 @@
+      responses:
+        207:
+          description: A multi-status code. The recipient needs to consult the contents
+            of the multi-status response body for further information about the success
+            or failure of the method execution.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BatchResponseDTO'
+        400:
+          description: There was a problem with the request. See the response body
+            for a more specific error message.
+          content: {}
+        401:
+          description: Access is unauthorized. The user needs to provide a valid bearer
+            token.
+          content: {}
+        403:
+          description: Access is forbidden.
+          content: {}
+        413:
+          description: The payload is larger than 1 MB.
+          content: {}
+        429:
+          description: Too many requests. The service is experiencing high loads.
+          content: {}
+        500:
+          description: An internal server error has occurred.
+          content: {}
+        503:
+          description: The service is currently unavailable. Clients should retry
+            at least three times using an exponential back-off strategy.
+          headers:
+            Retry-After:
+              description: Specifies the number of seconds to retry after.
+              schema:
+                type: integer
+          content: {}
+      x-codegen-request-body-name: body
+components:
+  schemas:
+    streamingMessage:
+      required:
+      - DATASET_ID
+      - IMS_ORG_ID
+      - SCHEMA_ID
+      - SCHEMA_VERSION
+      type: object
+      properties:
+        TENANT_ID:
+          type: string
+          description: The tenant ID ensures that the resources you create are properly
+            namespaced and contained within your organization.
+        SCHEMA_ID:
+          type: string
+          description: The ID of the schema that is created.
+        SCHEMA_VERSION:
+          type: string
+          description: The version of the schema that is created.
+        IMS_ORG_ID:
+          type: string
+          description: Your organization ID can be found under the integration
+            details in the Adobe Developer Console.
+        DATASET_ID:
+          type: string
+          description: The ID of the dataset that you want to publish to.
+        SOURCE_NAME:
+          type: string
+          description: An optional name for your source. If this value is missing,
+            the streaming message will automatically add the source ID from the streaming
+            connection definition.
+      description: The message that you want to ingest.
+      example:
+        header:
+          schemaRef:
+            id: https://ns.adobe.com/{TENANT_ID}/schemas/{SCHEMA_ID}
+            contentType: application/vnd.adobe.xed-full+json;version={SCHEMA_VERSION}
+          imsOrgId: '{IMS_ORG_ID}'
```

Comment:
Should be {ORG_ID}, but if you want to use a concrete example, feel free to use this one that I've used before: `5C1328435BF324E90A49402A@AdobeOrg`